modules/debts_api/app/sidekiq/debts_api/v0/form5655/vba_submission_job.rb

```diff
@@ -12,9 +12,15 @@ class V0::Form5655::VBASubmissionJob
 
     class MissingUserAttributesError < StandardError; end
 
-    sidekiq_retries_exhausted do |job, _ex|
+    sidekiq_retries_exhausted do |job, ex|
       user_uuid = job['args'][1]
       UserProfileAttributes.find(user_uuid)&.destroy
+      StatsD.increment("#{STATS_KEY}.exhausted")
```

Comment:
In order to not conflict with the existing "failure" stats, I'm adding a new "exhausted" stat. Let me know if we should instead modify the behavior from "Fail on each retry" to "Fail on exhaustion" though.

Reply:
This is great. I have a PR with something similar. WDYT about `retries_exhausted`. 

We should make it consistent I think:
https://github.com/department-of-veterans-affairs/vets-api/pull/18472/files

I don't really have a strong opinion, just don't collide/conflict like you said.

Reply:
Sounds good I'll update it to `retries_exhausted`